### PR TITLE
Add migration script for mirroring

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
This is able to replace ~75k support statements across 2000+ files, but I think the way to use this should be:
- First `node scripts/migrations/008-mirror.js --browsers chrome_android edge firefox_android opera opera_android safari_ios samsunginternet_android webview_android` for the cases that can be mirrored in all browsers, no complications to look into
- Then run things like `node scripts/migrations/008-mirror.js --browsers chrome_android webview_android` and `node scripts/migrations/008-mirror.js opera opera_android` and for smaller groups of things that will often be mirror-able
- Finally go through mirroring the individual browsers. This is the step where a lot of odd discrepancies might be discovered.

**Comment by queengooborg** [comment]
Could we set this to migration 009?  I have a pull request for a migration 008 that's waiting on your review!

**Comment by foolip** [comment]
I've renamed to 009. But also, do we need the numbers? We're missing some in the series, and will also delete scripts that have served their purpose.

File: scripts/migrations/009-mirror.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+import bcd from '../../index.js';
+import { walk } from '../../utils/index.js';
+import mirrorSupport from '../release/mirror.js';
+
+const dirname = fileURLToPath(new URL('.', import.meta.url));
+
+export const isMirrorEquivalent = (support, browsers) => {
+  // Make a (shallow) copy of the support data to work around
+  // https://github.com/mdn/browser-compat-data/issues/16471.
```
Comment (foolip) [inline scripts/migrations/009-mirror.js:22]: This was fixed, so I should simplify this again.

**Comment by foolip** [comment]
@queengooborg this is ready for review now.

**Review by queengooborg — APPROVED** [review]
I haven't tried running the script, but this looks great to me -- let's start migrating browsers to `mirror`!

**Pull request merged**